DOCS-1188 Run Synthetic Tests from Private Locations Edit (#13643)
DataDog/documentation: content/en/synthetics/private_locations/_index.md

    kubectl apply -f private-location-worker-deployment.yaml
    ```

> For OpenShift, you need to run the private location with the `anyuid` SCC in order for your browser test to run.

Reviewer: ```suggestion
For OpenShift, run the private location with the `anyuid` SCC. This is required for your browser test to run.
```
